docs/modules/ROOT/examples/hot-backup-external-azure.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: hazelcast.com/v1alpha1
+kind: HotBackup
+metadata:
+  name: hot-backup
+spec:
+  hazelcastResourceName: hazelcast
+  bucketURI: "azblob://operator-backup" <1>
```

Comment:
Can you add `/path/to/folder` like suffix to URI to give users an idea how they can give paths, same for AWS and GCP.